var/spack/repos/builtin/packages/py-fenics-dolfinx/package.py

```diff
@@ -49,20 +51,23 @@ class PyFenicsDolfinx(PythonPackage):
     depends_on("fenics-dolfinx@0.1.0", when="@0.1.0")
 
     depends_on("fenics-basix@main", type=("build", "link"), when="@main")
+    depends_on("fenics-basix@0.6.0:0.6", type=("build", "link"), when="@0.6.0:0.6")
     depends_on("fenics-basix@0.5.1:0.5", type=("build", "link"), when="@0.5.0:0.5")
     depends_on("fenics-basix@0.4.2", type=("build", "link"), when="@0.4.1")
     depends_on("fenics-basix@0.3.0", type=("build", "link"), when="@0.3.0")
     depends_on("fenics-basix@0.2.0", type=("build", "link"), when="@0.2.0")
     depends_on("fenics-basix@0.1.0", type=("build", "link"), when="@0.1.0")
 
     depends_on("py-fenics-ffcx@main", type="run", when="@main")
+    depends_on("py-fenics-ffcx@0.6.0:0.6", type="run", when="@0.6.0:0.6")
```

Comment:
```suggestion
    depends_on("py-fenics-ffcx@0.6", type="run", when="@0.6")
```
Simpler version of the same thing